src/libmongoc/src/mongoc/mongoc-async-cmd.c

```diff
@@ -35,9 +35,6 @@
 #include <mongoc/mongoc-stream-tls.h>
 #endif
 
-#undef MONGOC_LOG_DOMAIN
-#define MONGOC_LOG_DOMAIN "async"
```

Comment:
Suggest also removing the above unused `#include <mongoc/mongoc-log.h>`.

Defining `MONGOC_LOG_DOMAIN` before the include can override the logged domain. I initially thought the placement of this `#define` should instead be moved up, but this file does not appear to log.